tests/resources/complex_stack/suite.xml

```diff
@@ -51,6 +51,9 @@
                     <xpath function="case_name"/>
                 </text>
             </template>
+            <background>
+                <text>Regression for deprecated background tag</text>
+            </background>
```

Comment:
Added the regression to the stack test suite file. Doesn't really fit in, but didn't want to bother spinning up a new test and resource files. I can do that if you think it is worth it.